libeos-payg/manager.c

```diff
@@ -718,8 +718,7 @@ check_expired_cb (gpointer user_data)
   return G_SOURCE_CONTINUE;
 }
 
-/* Set the #EpgManager:expiry-time to
- * `MIN (G_MAXUINT64, MAX (@now, #EpgManager:expiry-time) + @span)` and set
+/* Set the #EpgManager:expiry-time to `MIN (G_MAXUINT64, @now + @span)` and set
```

Comment:
`@now` and `@span` parameters have since been renamed, these should change too.

Reply:
Oh, it took me a minute to realize that yes, it has changed some time in the past, not as part of this PR :smile:

I'm adding an extra commit updating the comment.